mantle/cmd/kola/console.go

```diff
@@ -78,4 +78,5 @@ func runCheckConsole(cmd *cobra.Command, args []string) {
 	if errors > 0 {
 		os.Exit(1)
```

Comment:
This should become a `errors.New(...)`.

Reply:
Heh, I did that at first and then changed my mind since this whole CLI is basically about printing errors above.  Dunno.

Reply:
OK done.